examples/examples-cpu/prefect/README.md

```diff
@@ -3,3 +3,4 @@
 The files in this directory were created to support these articles:
 
   * ["Scheduled Data Pipelines"](https://www.saturncloud.io/docs/deployments/data-pipelines)
+  * ["Fault-Tolerant Data Pipelines with Prefect Cloud"](https://www.saturncloud.io/docs/deployments/fault-tolerant-data-pipelines-with-prefect-cloud)
```

Comment:
Just note for future that I don't think this'll be the link

Reply:
ah yeah this'll change when we merge https://github.com/saturncloud/docs/pull/19, thanks for the reminder